client/weapons.lua

```diff
@@ -0,0 +1,98 @@
+weaponsList = {
+    "weapon_advancedrifle",
+    "weapon_appistol",
+    "weapon_assaultrifle",
+    "weapon_assaultrifle_mk2",
+    "weapon_assaultshotgun",
+    "weapon_assaultsmg",
+    "weapon_autoshotgun",
+    "weapon_bat",
+    "weapon_ball",
+    "weapon_battleaxe",
+    "weapon_bottle",
+    "weapon_bullpuprifle",
+    "weapon_bullpuprifle_mk2",
+    "weapon_bullpupshotgun",
+    "weapon_bzgas",
+    "weapon_carbinerifle",
+    "weapon_carbinerifle_mk2",
+    "weapon_combatmg",
+    "weapon_combatmg_mk2",
+    "weapon_combatpdw",
+    "weapon_combatpistol",
+    "weapon_compactlauncher",
+    "weapon_compactrifle",
+    "weapon_crowbar",
+    "weapon_dagger",
+    "weapon_dbshotgun",
+    "weapon_doubleaction",
+    "weapon_fireextinguisher",
+    "weapon_firework",
+    "weapon_flare",
+    "weapon_flaregun",
+    "weapon_flashlight",
+    "weapon_golfclub",
+    "weapon_grenade",
+    "weapon_grenadelauncher",
+    "weapon_gusenberg",
+    "weapon_hammer",
+    "weapon_hatchet",
+    "weapon_heavypistol",
+    "weapon_heavyshotgun",
+    "weapon_heavysniper",
+    "weapon_heavysniper_mk2",
+    "weapon_hominglauncher",
+    "weapon_knife",
+    "weapon_knuckle",
+    "weapon_machete",
+    "weapon_machinepistol",
+    "weapon_marksmanpistol",
+    "weapon_marksmanrifle",
+    "weapon_marksmanrifle_mk2",
+    "weapon_mg",
+    "weapon_microsmg",
+    "weapon_minigun",
+    "weapon_minismg",
+    "weapon_molotov",
+    "weapon_musket",
+    "weapon_nightstick",
+    "weapon_petrolcan",
+    "weapon_pipebomb",
+    "weapon_pistol",
+    "weapon_pistol50",
+    "weapon_pistol_mk2",
+    "weapon_poolcue",
+    "weapon_proxmine",
+    "weapon_pumpshotgun",
+    "weapon_pumpshotgun_mk2",
+    "weapon_railgun",
+    "weapon_revolver",
+    "weapon_revolver_mk2",
+    "weapon_rpg",
+    "weapon_sawnoffshotgun",
+    "weapon_smg",
+    "weapon_smg_mk2",
+    "weapon_smokegrenade",
+    "weapon_sniperrifle",
+    "weapon_snowball",
+    "weapon_snspistol",
+    "weapon_snspistol_mk2",
+    "weapon_specialcarbine",
+    "weapon_specialcarbine_mk2",
+    "weapon_stickybomb",
+    "weapon_stungun",
+    "weapon_switchblade",
+    "weapon_unarmed",
+    "weapon_vintagepistol",
+    "weapon_wrench",
+    "weapon_raypistol",
+    "weapon_raycarbine",
+    "weapon_rayminigun",
+    "weapon_stone_hatchet",
+    "weapon_ceramicpistol",
+    "weapon_navyrevolver",
+    "weapon_hazardcan", 
+    "weapon_gadgetpistol",
+    "weapon_militaryrifle",
+    "weapon_combatshotgun"
+}
```

Comment:
Instead of creating this `weapons.lua` file we could pull all of this information from `qb-core` utilizing the `QBShared.Weapons` table.

Reply:
still new to this not sure how to do that

Reply:
taken out